src/ListOfGoods.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ListOfGoods extends React.Component {
+  state = {
+    goods: this.props.goods,
```

Comment:
don't write props into state, props are needed to use them directly

https://reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state (look at the first point)

https://stackoverflow.com/a/50403930